AndroidClient/app/src/main/AndroidManifest.xml

```diff
@@ -1,108 +1,133 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.pixtory.app" >
+    package="com.pixtory.app">
+
     <uses-permission android:name="android.permission.SYSTEM_ALERT_WINDOW" />
     <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
     <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
-    <uses-permission android:name="android.permission.GET_TASKS"/>
+    <uses-permission android:name="android.permission.GET_TASKS" />
     <!-- required for setIsSyncable -->
     <uses-permission android:name="android.permission.INTERNET" />
-    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE"/>
-
+    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
 
     <!-- For recording -->
     <uses-permission android:name="android.permission.RECORD_AUDIO" />
```

Comment:
Do you need these permissions? In Android M there is a permission prompt that should be given. Please check Android M guidelines for permissions